src/app/project/beta/ontology-classes/ontology-class-instance/ontology-class-instance.component.ts

```diff
@@ -0,0 +1,108 @@
+import { Component, OnChanges, SimpleChanges } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { AppInitService } from 'src/app/app-init.service';
+import { OntologyService } from 'src/app/project/ontology/ontology.service';
+import { FilteredResources, SearchParams } from 'src/app/workspace/results/list-view/list-view.component';
+import { SplitSize } from 'src/app/workspace/results/results.component';
+
+@Component({
+    selector: 'app-ontology-class-instance',
+    templateUrl: './ontology-class-instance.component.html',
+    styleUrls: ['./ontology-class-instance.component.scss']
+})
+export class OntologyClassInstanceComponent implements OnChanges {
+
+    projectId: string;
+
+    classId: string;
+
+    instanceId: string;
+
+    searchParams: SearchParams;
+
+    // which resources are selected?
+    selectedResources: FilteredResources;
+
+    // display single resource or intermediate page in case of multiple selection
+    viewMode: 'single' | 'intermediate' | 'compare' = 'single';
+
+    splitSizeChanged: SplitSize;
+
+    constructor(
+        private _ais: AppInitService,
+        private _route: ActivatedRoute,
+        private _ontologyService: OntologyService
+    ) {
+
+        // parameters from the url
+        const projectCode = this._route.parent.snapshot.params.shortcode;
+
+        this.projectId = `${this._ais.dspAppConfig.iriBase}/projects/${projectCode}`;
+
+        this._route.params.subscribe(params => {
+            const iriBase = this._ontologyService.getIriBaseUrl();
+
+            const ontologyName = params['onto'];
+            const className = params['class'];
+
+            // get the resource class id from route
+            this.classId = `${iriBase}/ontology/${projectCode}/${ontologyName}/v2#${className}`;
+
+            this.instanceId = params['instance'];
+            if (this.instanceId) {
+                // single instance
+
+                if (this.instanceId === 'add') {
```

Comment:
In case of `this.instanceId` we have two options: in case of "add" the create-resource-instance-form is displayed. In case of "real" resource instance IRI/ID the single instance will be displayed. I thought to implement this view in the next iteration. 
Maybe there's a third option called "/conf" where a project admin can edit the resource class itself. 

I see, the chosen name "instanceId" is really confusing. I'll think about a better setup and will replace in the next branch.